projects/unix/tomviz.bundle.cmake

```diff
@@ -32,34 +38,75 @@ install(CODE
       -P ${CMAKE_CURRENT_LIST_DIR}/install_dependencies.cmake)"
   COMPONENT superbuild)
 
-if (qt_ENABLED AND NOT USE_SYSTEM_qt)
-  install(DIRECTORY
-    # install all qt plugins (including sqllite).
-    # FIXME: we can reconfigure Qt to be built with inbuilt sqllite support to
-    # avoid the need for plugins.
-    "${install_location}/plugins/"
-    DESTINATION "lib/tomviz"
-    COMPONENT superbuild
-    PATTERN "*.a" EXCLUDE
-    PATTERN "tomviz-${tomviz_version}" EXCLUDE
-    PATTERN "fontconfig" EXCLUDE
-    PATTERN "*.jar" EXCLUDE
-    PATTERN "*.debug.*" EXCLUDE
-    PATTERN "libboost*" EXCLUDE)
-  install(DIRECTORY "${install_location}/share/appdata"
-    DESTINATION "share"
-    USE_SOURCE_PERMISSIONS
-    COMPONENT superbuild)
-  install(DIRECTORY "${install_location}/share/applications"
-    DESTINATION "share"
-    USE_SOURCE_PERMISSIONS
-    COMPONENT superbuild)
-  install(DIRECTORY "${install_location}/share/icons"
-    DESTINATION "share"
-    USE_SOURCE_PERMISSIONS
-    COMPONENT superbuild)
+if (qt_ENABLED)
+  set(qt_plugins)
+  if (NOT USE_SYSTEM_qt)
+    file(GLOB qt_plugins ${install_location}/plugins/*)
```

Comment:
Then I propose simplifying the logic by defining it for both cases, and then removing the second set of not system Qt to reduce maintenance complexity. We can define the variable once for the superbuild version, and then treat it the same as system the rest of the time ideally.